doc/services/dsp/index.rst

```diff
@@ -0,0 +1,61 @@
+.. _dsp_api:
```

Comment:
This needs to match the ref in `overview.rst`.

```suggestion
.. _zdsp_api:
```

Reply:
Odd that it passed the docs build for me locally. Thanks for catching this.